docs/secp256k1/secp256k1.md

```diff
@@ -85,3 +85,8 @@ println(s"Verified with bouncy castle=${verified}")
 
 ```
 
+### Building libsecp256k1
+
+For information on how to build libsecp256k1 for various OS's see ACINQ's guide to building libsecp256k1 with the jni:
+
+https://github.com/ACINQ/bitcoin-lib/blob/master/BUILDING.md#building-jni-bindings-for-libsecp256k1
```

Comment:
We probably want to include some caveats to their directions, otherwise users will be running things like:
```
cp ./.libs/libsecp256k1.so ../src/main/resources/fr/acinq/native/Linux/x86_64/
```